bears/hypertext/BootLintBear.py

```diff
@@ -27,9 +27,9 @@ class BootLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         bootlint_ignore: typed_list(str)=[]):
+                         lint_ignore: typed_list(str)=[]):
         """
-        :param bootlint_ignore: List of checkers to ignore.
+        :param lint_ignore: List of checkers to ignore.
         """
-        ignore = ','.join(part.strip() for part in bootlint_ignore)
+        ignore = ','.join(part.strip() for part in lint_ignore)
```

Comment:
I'm not sure if this is a good thing, I couldn't ignore something from some linter but not from the other... not sure maybe it does make sense. Feel like ths might need discussion. Has it been discussed so far?